lessons/beginners/class/index.md

```diff
@@ -149,7 +153,7 @@ aby se nepletly s „normálními“ hodnotami.
 
 V těle třídy můžeš definovat metody, které vypadají
 úplně jako funkce – jen mají první argument `self`.
```

Comment:
I would like to ask about `self`. Here it is an *argument* and later in the materials it is a *parametr*. I am still not 100% sure about the difference between these two terms. Or are the rules for when to use which one of them different for functions and methods?

Reply:
They have precise meanings, bit I (and other programmers) unfortunately often mix them up.
It's like the words *zásuvka*/*zástrčka*, both in what they mean and in how people mix them up :)

* A *parameter* is seen in the function definition. It is the variable *name*.
* An *argument* is the *value* passed to the function when the function is called.

```suggestion
úplně jako funkce – jen mají první parametr `self`.
```